akka-cluster/src/main/scala/akka/cluster/Cluster.scala

```diff
@@ -356,6 +356,15 @@ class Cluster(val system: ExtendedActorSystem) extends Extension {
     else
       clusterDaemons ! InternalClusterAction.AddOnMemberRemovedListener(callback)
   }
+
+  /**
+   * Generate the actor path by replacing the Address in the RootActor Path for the given
+   * ActorRef with the cluster's `selfAddress`, unless an address is already included
+   */
+  def pathOf(actorRef: ActorRef): ActorPath = {
+    ActorPath.fromString(actorRef.path.toStringWithAddress(selfAddress))
+  }
```

Comment:
Hmm, but then it would not be convenient (i.e. what you wrote) or not obvious that it uses the Cluster extension—and that is not even possible when doing it in akka-actor.